### ISSUE TITLE
Are side transitions supported

### ISSUE THREAD

**Issue by maziggy** [tomvandeneede/p2pp_s3d#2]
I added 

;P2PP SIDEWIPELOC=X254
;P2PP SIDEWIPEMINY=45
;P2PP SIDEWIPEMAXY=195
;P2PP SIDEWIPECORRECTION=1.0

to my S3d profile but the resulting code says:

; process Process1(1)
;---- T1 TOOLCHANGE -----
; --------------------------------------------------
; --- P2PP WIPE SEQUENCE START  FOR 120.00mm
; --- DELTA = -0.20
; --------------------------------------------------
G1 E-0.8
G1 X-9998.000 Y9484.129 F3000
G1 Z3.00 F10800
G1 E0.8
G1 F3000
G1 X-9998.0 Y9590.754 E120.0  ;inter -9998.000 9998.000
;----------------------------------
; P2 PING 124 COMMAND AT 83163.098mm
O31 D47a26d8c
;----------------------------------
; -------------------------------------
G1 E-0.8
G1 Z3.20 F10800
G0 X160.587 Y152.442 F4000
G1 E0.8
; --- P2PP WIPE SEQUENCE END DONE
; -------------------------------------
;-- P2PP/S3D -- removed [Toolchange handled by Palette] - T1

Thanks!

**Comment by tomvandeneede** [comment]
S3D currently does not support sidewipe as the mechanism to put the prime pillar outside of the bed is not supported in S3D as far as I know

Have you defined your bedside correctly?  
I am looking for strokeXoverride and strokeYoverride in the gcode  as well as originOffsetXoverride and originOffsetYoverride
The bedside and beg origin are used to calculate the side wipe location.

**Comment by maziggy** [comment]
I wrote my a python script which converts to the purging parts to the format I need. Works like charm ;)

**Comment by tomvandeneede** [comment]
sounds good! 

if there is a consistent config way I can put it in my code as well but I found that S#d config is a lot more complex when it comes to just adding a few config lines and also for retrieving the actual config parameters
It is fairly easy to make something that works but if you want to make it generally usable there are a lot of obstacles

**Comment by maziggy** [comment]
Yeah, right. My script is also a “custom workaround”. Had to insert some remarks into the start code that I can parse.
